spec/features/user_votes_on_review_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+feature "User votes on a Review",
+%q(
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use `%q` only for strings that contain both single quotes and double quotes.